docs/source/tutorials/install.rst

```diff
@@ -0,0 +1,101 @@
+.. _run-webpush-channels-python:
+
+Using the Python package
+========================
+
+System requirements
+-------------------
+
+Depending on the platform and chosen configuration, some libraries or
+extra services are required.
+
+The following commands will install necessary tools for cryptography
+and Python packaging like `Virtualenv <https://virtualenv.pypa.io/>`_.
+
+Linux
+'''''
+
+On Debian / Ubuntu based systems::
+
+    apt-get install libffi-dev libssl-dev python-dev python-virtualenv
+
+On RHEL-derivatives::
+
+    dnf install libffi-devel openssl-devel python-devel python-virtualenv
+
+OS X
+''''
+
+Assuming `brew <http://brew.sh/>`_ is installed:
+
+::
+
+    brew install libffi openssl pkg-config python
+
+    pip install virtualenv
+
+
+Quick start
+-----------
+
+By default, for convenience, *webpush-channels* persists the subscriptions and channel
+registartions in a **volatile** memory backend. On every restart, the server
+will lose its data, and multiple processes are not handled properly.
+
+But it should be enough to get started!
+
+
+Create a Python isolated environment:
+
+::
+
+    virtualenv env/
+    source env/bin/activate
+
+Install kinto:
+
+::
+
+    pip install kinto
```

Comment:
Since the last release, Kinto 6.0.0 is only compatible with Python 3.5 so either we need to make webpush-channels compatible with Python3.5 or we need to do ``pip install kinto<6``